idd/Energy+.idd.in

```diff
@@ -18385,6 +18385,9 @@ RoomAirModelType,
       \key UnderFloorAirDistributionExterior
       \note (2-Node UFAD model for exterior zones)
       \note needs RoomAirSettings:UnderFloorAirDistributionExterior object referencing this Zone
+      \key AirflowNetwork
+      \note (RoomAir modeling using AirflowNetwork)
```

Comment:
@lgu1234 Notes are at the field level, not the key level, so this should be something like "AirflowNetwork = RoomAir modeling using AirflowNetwork.  Several other notes in this field need a similar change.